src/index.d.ts

```diff
@@ -1 +0,0 @@
-export default function invariant(condition: any, message?: string): void
```

Comment:
@alexreardon should the return type be `void | never`?

ref: 
- https://www.typescriptlang.org/docs/handbook/basic-types.html#never
- https://stackoverflow.com/a/49435648

Reply:
I see that asserts condition is the alternate for `void|never`. Can ignore this one.